decidim-core/app/cells/decidim/author_cell.rb

```diff
@@ -55,8 +55,28 @@ def withdraw
       render
     end
 
+    def perform_caching?
+      true
+    end
+
     private
 
+    def cache_hash
+      hash = []
+
+      hash.push(I18n.locale)
+      hash.push(model.cache_key_with_version) if model.respond_to?(:cache_key_with_version)
+      hash.push(current_user.present? ? 1 : 0)
+      hash.push(commentable? ? 1 : 0)
+      hash.push(endorsable? ? 1 : 0)
+      hash.push(actionable? ? 1 : 0)
+      hash.push(withdrawable? ? 1 : 0)
+      hash.push(flaggable? ? 1 : 0)
+      hash.push(profile_path? ? 1 : 0)
+      hash.push(current_user.try(:id)) if current_user.present?
```

Comment:
Ditto, does it make sense caching this cell with the current_user.id? Also, any reason for using three different ways of doing this? Can we always do the same approach if it's necessary to do so?

Reply:
yes, it does make sense to cache with current_user id. The author Cell also renders the user card that has follow information. So we need to know if an user follows another one of not. 

I will change the current_user to have consistency.